### PR TITLE
fixing the integration test in order not to get gRPC exceptions

### PR DESCRIPTION
**Pull request by davidrabinowitz**

### CODE REVIEW

**Comment by kmjung** [comment]
Can you elaborate on what gRPC exceptions you're seeing?

File: connector/src/test/scala/com/google/cloud/spark/bigquery/it/SparkBigQueryEndToEndITSuite.scala [hunk 1]
```diff
@@ -66,7 +66,7 @@ class SparkBigQueryEndToEndITSuite extends FunSuite
       "The year in which this corpus was published."))))
   private val LARGE_TABLE = "bigquery-public-data.samples.natality"
   private val LARGE_TABLE_FIELD = "is_male"
-  private val LARGE_TABLE_NUM_ROWS = 137826763L
+  private val LARGE_TABLE_NUM_ROWS = 33271914L
```
Comment (Gaurangi94) [inline connector/src/test/scala/com/google/cloud/spark/bigquery/it/SparkBigQueryEndToEndITSuite.scala:69]: how is this change fixing GRPC exception?

Comment (Gaurangi94) [reply connector/src/test/scala/com/google/cloud/spark/bigquery/it/SparkBigQueryEndToEndITSuite.scala:69]: Resolved offline - Details can be found at b/154249285

**Comment by davidrabinowitz** [comment]
Hi @kmjung these are the exceptions I sent you in private last week

**Pull request merged**